Corona-Warn-App/src/main/java/de/rki/coronawarnapp/tracing/ui/details/items/periodlogged/PeriodLoggedBox.kt

```diff
@@ -34,14 +34,9 @@ class PeriodLoggedBox(
     }
 
     data class Item(
-        val activeTracingDaysInRetentionPeriod: Int,
         val tracingStatus: GeneralTracingStatus.Status
     ) : DetailsItem {
 
-        fun getRiskActiveTracingDaysInRetentionPeriodLogged(context: Context): String = context.getString(
-            R.string.risk_details_information_body_period_logged_assessment_under_14_days
```

Comment:
this string too

Reply:
No please, this is part of my implementation. These strings were added some time last week. :)